src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -326,6 +335,34 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
         return true;
     }
 
+    private ScheduleRunConfiguration getScheduleRunConfiguration(Boolean isRunUnmetered) {
+    	 ScheduleRunConfiguration configuration = new ScheduleRunConfiguration();
+    	 if(isRunUnmetered != null && isRunUnmetered) {
+    		 configuration.setBillingMethod(BillingMethod.UNMETERED);
+    	 } else {
+    		 configuration.setBillingMethod(BillingMethod.METERED);
+    	 }
+    	 
+    	 // set a bunch of other default values as Device Farm expect these
+    	 configuration.setAuxiliaryApps(new ArrayList<String>());
```

Comment:
I actually run into this issue while building canaries where aws device farm stack (lower down) expects these values to be present . Otherwise the tests crashed.